test/suite/test_alter01.py

```diff
@@ -176,5 +176,4 @@ def test_alter01_access(self):
                         self.verify_metadata(access_str)
                         self.verify_metadata(cache_str)
 
-if __name__ == '__main__':
-    wttest.run()
+
```

Comment:
As discussed IRL - would be better to avoid a second blank line here.

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10063/commits/93b3582cb7e017ca29b1bf13af7053566d4db825